### PR TITLE
Sync script

### PR DESCRIPTION
**Pull request by MattF-NSIDC**

### CODE REVIEW

File: admin/before-workshop.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ index: -99
 - [x] Pre-populate the "Dataset Ideas" section of GH Discussions with multiple posts,
       each detailing one "fallback" dataset for groups that have trouble picking one.
 - [x] Create a space for daily run-of-show documents (e.g. Google Docs)
+- [ ] symlink / copy setup/sync script from repository location to
```
Comment (MattF-NSIDC) [inline admin/before-workshop.md:16]: I think we can mark this checked :)
```suggestion
- [x] symlink / copy setup/sync script from repository location to
```

File: content/exercises/data-inspection-with-jupyterlab/index.md [hunk 2]
```diff
@@ -14,6 +14,13 @@ Pick someone to "drive" with a screen share. Please rotate this responsibility!
 ## 🔍 Inspect some data
 
 * Create a new Jupyter Notebook to inspect your group's chosen dataset.
+
+::: {.callout-note}
+Remember that you uploaded example data from your group's chosen dataset to
+`/shared/public/QGreenland-Researcher-Workshop/group-{your-group-number}` in
```
Comment (MattF-NSIDC) [inline content/exercises/data-inspection-with-jupyterlab/index.md:20]:
```suggestion
`~/shared-public/QGreenland-Researcher-Workshop/group-{your-group-number}` in
```

File: content/exercises/getting-started-with-jupyterlab.md [hunk 3]
```diff
@@ -34,20 +43,27 @@ familiar with the JupyterLab interface.
     ![Use default config](/_media/xfce_first_launch.png)
 
 * Open QGIS from the desktop icon.
-* Open QGreenland from `shared/QGreenland`.
+* Open QGreenland from `/home/jovian/qgis-data/QGreenland_v3.0.0alpha2/`.
```
Comment (MattF-NSIDC) [inline content/exercises/getting-started-with-jupyterlab.md:46]: Maybe note that if this directory isn't present, to go back to the sync step?

**Comment by MattF-NSIDC** [comment]
LGTM

**Review by trey-stafford — APPROVED** [review]

**Pull request merged**